include/constants.h

```diff
@@ -4,6 +4,7 @@ namespace reinforcement_learning {  namespace name {
       const char *const  APP_ID                  = "appid";
       const char *const  MODEL_SRC               = "model.source";
       const char *const  MODEL_BLOB_URI          = "model.blob.uri";
+      const char* const  MODEL_HTTP_API_HOST    = "model.http.api.host";
```

Comment:
The is an HTTP URI not a host. The host is the domain (cognitiveservices.com) not the entire URL. I would be fine with reusing MODEL_BLOB_URI.